### PR TITLE
Vali/versioned txn

### PR DESCRIPTION
**Pull request by alittlezz**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -29,7 +29,7 @@ typing-extensions = ">=4.2.0"
 cachetools = "^4.2.2"
 types-cachetools = "^4.2.4"
 websockets = "^10.3"
-solders = "^0.10.0"
+solders = { git = "https://github.com/hubbleprotocol/solders.git", branch = "vali/versioned-txn" }
```
Comment (michaelhly) [inline pyproject.toml:32]: cc @kevinheavey, can you take a look at https://github.com/kevinheavey/solders/pull/32

**Comment by kevinheavey** [comment]
Btw @alittlezz we're happy to just use the `solders` transaction classes directly and remove `solana.transaction` if that makes this easier

File: pyproject.toml (hunk 1 above)
Comment (alittlezz) [reply pyproject.toml:32]: Hey, thanks for looking at this! The combination of these two PRs is enough for VersionedTransaction to work (both legacy and V0). Unfortunately the code breaks the API (it should use a `union[Transaction, VersionedTransaction]`) and has some hacky parts as the interface between Message/VersionedMessage and Transaction/VersionedTransaction has changed a bit. But hopefully this serves as a baseline for starting the work on this feature. At the moment I don't really have capacity to make this nicer, it's mostly a solution for who really needs versioned transactions in solana right now.

Also I cannot figure out how to format the files corretly, have used `black` with line limit set to 120 but it ignores the pylint ignore comments and formats some dependencies weirdly.

Comment (michaelhly) [reply pyproject.toml:32]: run ‘make format’ for formatting

**Comment by kevinheavey** [comment]
Closed by #334

**Pull request closed without merging**